test/toAdditive.lean

```diff
@@ -183,6 +183,19 @@ def IsUnit [Mul M] (a : M) : Prop := a ≠ a
 theorem isUnit_iff_exists_inv [Mul M] {a : M} : IsUnit a ↔ ∃ _ : α, a ≠ a :=
   ⟨fun h => absurd rfl h, fun ⟨_, hab⟩ => hab⟩
 
+/-! Test that `@[to_additive]` correctly translates auxilliary declarations that do not have the
```

Comment:
According to my firefox spellchecker:

```suggestion
/-! Test that `@[to_additive]` correctly translates auxiliary declarations that do not have the
```

Could you please expand this docstring? Which of the following `to_additive` calls generates such an auxiliary declaration?